src/share/classes/com/sun/javatest/regtest/agent/JUnitRunner.java

```diff
@@ -170,4 +175,21 @@ private static void runWithJUnitPlatform(Class<?> mainClass) throws Exception {
             throw new Exception(JUNIT_NO_DRIVER, ex);
         }
     }
+
+    public static class PrintingListener implements TestExecutionListener {
+        @Override
+        public void executionFinished(TestIdentifier identifier, TestExecutionResult result) {
+            System.out.println(identifier.getDisplayName() + ": " + result.getStatus());
```

Comment:
Which `System` stream is better? `out`, or `err` ?

Reply:
Maybe `PrintingListener` should have a constructor which takes a `PrintStream` or `PrintWrite`, so that we only get to reference the `System` stream once, when the listener is created...

Reply:
Using `System.err` via a constructor parameter since https://github.com/openjdk/jtreg/pull/127/commits/7adedc204ed22192cd25a9ed42dad4f0af20249c